### DISCUSSION TITLE
How to Check String Condition in java

### DISCUSSION THREAD

**Discussion by NaveedAzmi** [community/community#37921] [Other Feature Feedback, Questions, & Ideas]
Hello All Respected Dear

Need your Help, Please Kindly, I am java Beginner **self-taught Programmer** in java and I want to know how to Check String Condition.
I shall Thankful to you

**Comment by hongducdev** [comment]
You can read more at: https://www.javatpoint.com/string-comparison-in-java
It is a useful site for people who need to learn Java and I am reading it too. Hope it can be of help to you. Good luck.

**Reply by NaveedAzmi** [reply]
Thank You, Dear hongduccodedao
